src/colmap/controllers/incremental_mapper.cc

```diff
@@ -552,7 +552,7 @@ void IncrementalMapperController::TriangulateReconstruction(
     LOG(INFO) << StringPrintf("Triangulating image #%d (%d)", image_id, i);
     const size_t num_existing_points3D = image.NumPoints3D();
     LOG(INFO) << "=> Image sees " << num_existing_points3D << " / "
-              << image.NumObservations() << " points";
+              << image.NumPoints2D() << " points";
```

Comment:
The # observations is more meaningful, as it is the upper bound of visible points in the image, but it's not critical if not easy to do here.

Reply:
Done.